Description: [orcus] use the foreground color for solid fill
  Ref: https://gitlab.com/orcus/orcus/commit/f821995022df8dd1e580dd22cf131584b2b1ac4f
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Bug: https://bugs.documentfoundation.org/show_bug.cgi?id=115931
Bug-Ubuntu: https://launchpad.net/bugs/1750982
Forwarded: no

--- a/sc/source/filter/orcus/interface.cxx
+++ b/sc/source/filter/orcus/interface.cxx
@@ -839,7 +839,7 @@ void ScOrcusStyles::fill::applyToItemSet
         return;
     }
 
-    rSet.Put(SvxBrushItem(maBgColor, ATTR_BACKGROUND));
+    rSet.Put(SvxBrushItem(maFgColor, ATTR_BACKGROUND));
 }
 
 ScOrcusStyles::protection::protection():
